### PR TITLE
Updates to "the Matrix".

### PR DESCRIPTION
**Pull request by erictune**

### CODE REVIEW

**Comment by erictune** [comment]
@davidopp this is a step prior to the decision tree we talked about.

File: docs/devel/writing-a-getting-started-guide.md [hunk 1]
```diff
@@ -36,11 +37,12 @@ the combination of all these things needed to run on a particular type of infras
 which is similar to the one you have planned, consider improving that one.
 
 
-Distros fall into two categories:  
+Distros fall into two categories:
   - **versioned distros** are tested to work with a particular binary release of Kubernetes.  These
     come in a wide variety, reflecting a wide range of ideas and preferences in how to run a cluster.
-  - **development distros** are tested work with the latest Kubernetes source code.  But, there are
-    relatively few of these and the bar is much higher for creating one.
+  - **development distros** are tested work with the latest Kubernetes source code.
```
Comment (davidopp) [inline docs/devel/writing-a-getting-started-guide.md]: You chopped off the "But, there are" at the end of this line

File: docs/getting-started-guides/README.md [hunk 2]
```diff
@@ -28,38 +28,38 @@ For the easiest "kick the tires" experience, please try the [local docker](docke
...
-Bare-metal           | custom       | Ubuntu | flannel     | [docs](ubuntu.md)                            |             | Community (@resouer @WIZARD-CXY)       | use k8s version 0.19.3
-Local                |              |        | _none_      | [docs](locally.md)                           |             | Community (@preillyme)      |
-libvirt/KVM          | CoreOS       | CoreOS | libvirt/KVM | [docs](libvirt-coreos.md)                    |             | Community (@lhuard1A)       |
-oVirt                |              |        |             | [docs](ovirt.md)                             |             | Community (@simon3z)        |
-Rackspace            | CoreOS       | CoreOS | flannel     | [docs](rackspace.md)                         |             | Community (@doublerr)       | use k8s version 0.18.0
+IaaS Provider        | Config. Mgmt | OS     | Networking  | Docs                                              | Conforms | Support Level
+-------------------- | ------------ | ------ | ----------  | ---------------------------------------------     | ---------| ----------------------------
```
Comment (davidopp) [inline docs/getting-started-guides/README.md]: Where are you planning to move the info about last version tested with each config?

File: docs/devel/writing-a-getting-started-guide.md (hunk 1 above)
Comment (erictune) [inline docs/devel/writing-a-getting-started-guide.md]: Thanks, fixed.

File: docs/getting-started-guides/README.md (hunk 2 above)
Comment (erictune) [inline docs/getting-started-guides/README.md]: I am not.  People can put this in the getting-started-guide if they really want to.

This was useful when kubernetes was changing rapidly.
However, it was not updated reliably.  Also, simply using
a particular binary version does not assure users that
the distribution provides a conformant implementation.
The conformance test does provide that, and I'm hoping
that more distros will use that in the future.
Therefore, I've deleted the Notes column in favor of the
Conforms column.

**Comment by davidopp** [comment]
LGTM

**Pull request merged**